temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using `pdb` (Python Debugger) in production code. If this is for debugging purposes, it should be removed before finalizing the PR.
```
Remove the line `import pdb`
```